.vscode/launch.json

```diff
@@ -295,6 +295,22 @@
         "services/validation/test/files/single"
       ]
     },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Validation service (standard-json file input)",
```

Comment:
We can comment out this for now since we don't support standard json so it is a bit confusing.